### PR TITLE
Add identity systems

### PR DESCRIPTION
**Pull request by blegat**
One system to rule them all

### CODE REVIEW

File: src/continuous.jl [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+struct ContinuousIdentitySystem <: AbstractContinuousSystem
+    statedim::Int
+end
+statedim(s::ContinuousIdentitySystem) = s.statedim
+inputdim(s::ContinuousIdentitySystem) = 0
+
+"""
+    ConstrainedContinuousIdentitySystem <: AbstractContinuousSystem
+
+Trivial identity continuous-time system of the form
```
Comment (schillic) [inline src/continuous.jl]: Maybe add "with state constraints" in line with `ConstrainedLinearContinuousSystem`?
(same for the discrete system)

File: test/continuous.jl [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+        s = ContinuousIdentitySystem(sd)
+        @test statedim(s) == sd
+        @test inputdim(s) == 0
+    end
+end
+
+@testset "Continuous constrained identity system" begin
+    for sd in 1:3
+        X = Line([1., -1], 0.) # line x = y
+        s = ConstrainedContinuousIdentitySystem(sd, X)
```
Comment (schillic) [inline test/continuous.jl]: Should the constraint not be of dimension `sd`? (Maybe I do not understand how this works.)
(same for the discrete system)

Comment (blegat) [reply test/continuous.jl]: Indeed ^^

File: test/continuous.jl [hunk 3]
```diff
@@ -7,12 +7,10 @@
 end
 
 @testset "Continuous constrained identity system" begin
-    for sd in 1:3
-        X = Line(ones(sd), 0.)
```
Comment (schillic) [inline test/continuous.jl]: Sorry, I realized this only later: You can use `Singleton(ones(sd))` for this.

File: test/continuous.jl (hunk 2 above)
Comment (mforets) [reply test/continuous.jl]: yes, it was my fault -- i added a 2D `Line` but it was meant to be hyperplanes. anyways, `Singleton` is ok.

**Pull request merged**